pyproject.toml

```diff
@@ -53,25 +53,25 @@ pytest = "*"
 pytest-cov = "*"
 pylama = "*"
 requests-mock = "*"
-black = { version = "21.10b0", allow-prereleases = true }
+black = "^22.1.0"
 mypy = "*"
 # The following dependencies are used for docs generation when run locally or in Docker
 # (e.g. poetry install)
-sphinx = "^1"
-sphinx_rtd_theme = "^0.4"
+sphinx = "^4"
+sphinx_rtd_theme = "^1.0"
 sphinxcontrib_napoleon = "^0.7"
 jupyter = "^1"
-nbsphinx = "^0.5"
+nbsphinx = "^0.8"
 pygments = "^2"
-sphinx-issues = "^1.2"
+sphinx-issues = "^3.0"
 
 # for docs only
-## this needs to be fixed, poetry is giving issues due to nornir being beta
-nornir-jinja2 = "0.1.2"
+nornir-jinja2 = "0.2.0"
 nornir-utils = "0.1.2"
 nornir-napalm = "0.1.2"
-# due to https://github.com/google/textfsm/issues/63
```

Comment:
@ktbyers This is fixed in version 1.1.2 right?

Reply:
Yes, that is fixed in >= 1.1.2 (so we should require at least textfsm version 1.1.2 in the dev-dependencies).

Reply:
Yes I did on line 74
`textfsm = "^1.1.2"`

Reply:
I should have made the comment on line. 74. I had in mind it shows 2 lines before and after